src/components/select/simple-select/index.d.ts

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import Option from '../option';
+
+export interface SimpleSelectProps {
+  /** The selected value(s), when the component is operating in controlled mode */
+  value: string | object;
```

Comment:
Should these have `?` as they're not required

Reply:
fixed